UPGRADE-5.3.md

```diff
@@ -0,0 +1,8 @@
+UPGRADE FROM 5.2 to 5.3
+=======================
+
+PropertyInfo
+------------
+
+* Deprecated `$builtinType` string type (first argument of `Type::__construct`
```

Comment:
I don't think we need to deprecate accepting a string. Better reduce the cost for the community.